src/ecommerce/components/layout/Layout.jsx

```diff
@@ -4,8 +4,8 @@ import { InstantSearch } from 'react-instantsearch-dom'
 import { instantMeiliSearch } from '@meilisearch/instant-meilisearch'
 
 const searchClient = instantMeiliSearch(
-  process.env.NEXT_PUBLIC_MEILI_HOST_NAME,
-  process.env.NEXT_PUBLIC_MEILI_API_KEY
+  "https://ms-336e328523ff-106.lon.meilisearch.io",
```

Comment:
@oluademola we need to update these values in Vercel. so these should still be environment variables.